guidelines/sc/20/reading-level.html

```diff
@@ -4,7 +4,7 @@ <h4>Reading Level</h4>
    
    <p class="conformance-level">AAA</p>
    
-   <p>When text requires reading ability more advanced than the <a>lower secondary education level</a> after removal of proper names and titles, <a>supplemental content</a>, or a version that does not require reading ability more advanced than the lower
+  <p>When <a>text</a> requires reading ability more advanced than the <a>lower secondary education level</a> after removal of proper names and titles, <a>supplemental content</a>, or a version that does not require reading ability more advanced than the lower
```

Comment:
I agree that linking this to the definition of text has the potential to except something conveyed through an Image of Text from having its reading level assessed.

```suggestion
  <p>When text requires reading ability more advanced than the <a>lower secondary education level</a> after removal of proper names and titles, <a>supplemental content</a>, or a version that does not require reading ability more advanced than the lower
```